examples/webui-rest/main.c

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2022 Cesanta Software Limited
+// All rights reserved
+//
+// REST basics example
+// It implements the following endpoints:
+//    /api/f1 - respond with a simple mock result
+//    /api/sum - respond with the result of adding two numbers
+//    any other URI serves static files from s_root_dir
+// Results are JSON strings
+
+
+#include "mongoose.h"
+
+static const char *s_http_addr = "http://localhost:8000";  // HTTP port
+static const char *s_root_dir = "web_root";
+
+static void fn(struct mg_connection *c, int ev, void *ev_data, void *fn_data) {
+  if (ev == MG_EV_HTTP_MSG) {
+    struct mg_http_message *hm = (struct mg_http_message *) ev_data;
+    if (mg_http_match_uri(hm, "/api/f1")) {
+      mg_http_reply(c, 200, "Content-Type: application/json\r\n",
+                    "{\"result\": %d}\n", 123);
```

Comment:
```c
mg_http_reply(c, 200, "...headers...",  "{%Q:%d}\n", "result", 123);
```

Reply:
Yes, this one I first though it was nice to show %Q later, but it is better to change it